.circleci/config.yml

```diff
@@ -2,33 +2,84 @@ version: 2.1
 
 orbs:
   # Using 1.0 of our Orbs means it will use the latest 1.0.x version from https://github.com/wordpress-mobile/circleci-orbs
-  ios: wordpress-mobile/ios@1.0 
+  ios: wordpress-mobile/ios@1.0
+
+# YAML anchors for some common/repeated values
+x-common-params:
+  - &xcode-version "12.2.0"
```

Comment:
Would Xcode 12.4 ( work as well? Or is there some compatibility issue with the rest of the tooling?

Reply:
Yeah, good call. Seems to work great, so let's roll with it!